common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -3,21 +3,46 @@ metadata:
     display_name: Checkboxes
```

Comment:
Yes, that is what it does. I will want @explorerleslie's opinion here.

Leslie, do we want the compound hinting to appear in the default "new problem" yaml for checkbox, multiplechoice, numericalresponse, optionresponse, and string_response problems in Studio?

I'll have a sandbox where you can see this as soon as this PR is rebased, so OK if you want to punt your decision until you can see it in action.